### PR TITLE
TIP-843: Move connector component classes to Enrichment bounded context

### PR DESCRIPTION
**Pull request by arnolanglade**
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | yes
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by arnolanglade** [comment]
1 behat scenario failed but it passed locally.

File: src/Akeneo/Pim/Enrichment/Component/spec/Product/Connector/Processor/Denormalizer/ProductModelLoaderProcessorSpec.php [hunk 1]
```diff
@@ -16,10 +16,11 @@ function let(StepExecution $stepExecution, ProductModelRepositoryInterface $repo
         $this->setStepExecution($stepExecution);
         $repository->getIdentifierProperties()->willReturn(['code']);
     }
-    
+
     function it_is_initializable()
     {
-        $this->shouldHaveType(ProductModelLoaderProcessor::class);
+        $this->shouldHaveType(
+           ProductModelLoaderProcessor::class);
```
Comment (LaurentPetard) [inline src/Akeneo/Pim/Enrichment/Component/spec/Product/Connector/Processor/Denormalizer/ProductModelLoaderProcessorSpec.php]: Useless line break

File: src/Akeneo/Pim/Enrichment/Component/spec/Product/Connector/Processor/Denormalizer/FindProductToImportSpec.php [hunk 2]
```diff
@@ -23,7 +23,8 @@ function let(
 
     function it_is_initializable()
     {
-        $this->shouldHaveType(FindProductToImport::class);
+        $this->shouldHaveType(
+            FindProductToImport::class);
```
Comment (LaurentPetard) [inline src/Akeneo/Pim/Enrichment/Component/spec/Product/Connector/Processor/Denormalizer/FindProductToImportSpec.php]: Same

File: CHANGELOG-3.0.md [hunk 3]
```diff
@@ -412,4 +412,69 @@
...
+- Move `Pim\Component\Connector\Processor\Denormalization\ProductModelLoaderProcessor` to `Akeneo\Pim\Enrichment\Component\Product\Connector\Processor\Denormalizer\ProductModelLoaderProcessor`
+- Move `Pim\Component\Connector\Processor\Denormalization\ProductModelProcessor` to `Akeneo\Pim\Enrichment\Component\Product\Connector\Processor\Denormalizer\ProductModelProcessor`
+- Move `Pim\Component\Connector\Processor\Denormalization\ProductProcessor` to `Akeneo\Pim\Enrichment\Component\Product\Connector\Processor\Denormalizer\ProductProcessor`
+- Move `Pim\Component\Connector\Processor\Normalization\ProductProcessor` to `Akeneo\Pim\Enrichment\Component\Product\Connector\Processor\Normalization\ProductProcessor`
+- Move namespace`Pim\Component\Connector\Analyzer` to `Akeneo\Pim\Enrichment\Component\Product\Connector`
+- Move `Pim\Component\Connector\Writer\File\ProductColumnSorter` to `Akeneo\Pim\Enrichment\Component\Product\Connector\ProductColumnSorter`
+- Move `Pim\Component\Connector\Reader\Database\GroupReader` to `Pim\Component\Connector\Reader\Database\GroupReader`
```
Comment (jjanvier) [inline CHANGELOG-3.0.md]: nope :)

File: CHANGELOG-3.0.md [hunk 4]
```diff
@@ -412,4 +412,69 @@
...
+- Move `Pim\Component\Connector\Reader\Database\ProductReader` to `Pim\Component\Connector\Reader\Database\ProductReader`
```
Comment (jjanvier) [inline CHANGELOG-3.0.md]: nope :)

**Pull request merged**